packages/reactstrap-validation-date/styles.scss

```diff
@@ -1,3 +1,150 @@
-.popover.popover-calendar, .popover.popover-calendar-range {
```

Comment:
i believe border radius on the input needs to be changed to look like a reactstrap input.

Reply:
Yeah the selector's for `first-of-type` and `last-of-type` are not working properly. Updating them now.